proposals/0012-f3-deals-reliability-ux.md

```diff
@@ -0,0 +1,159 @@
+# Project F3 -- Filecoin storage deals reliability and UX
+
+Authors: @raulk, @dirkmc, @nonsense.
+
+Initial PR: https://github.com/protocol/web3-dev-team/pull/12.
+
+## Purpose & impact 
+
+#### Background & intent
+
+_Describe the desired state of the world after this project? Why does that matter?_
+
+Filecoin aims to become the storage layer for the decentralised future.
+The web3/dweb communities are excited, engaged, and traction is on the uptick everyday.
+
+Mining, consensus, onboarding storage capacity, and proving sectors, are all fully functional processes in the Filecoin Network, and namely in the reference implementation: [Lotus](https://github.com/filecoin-project/lotus). However, the object of this proposal, **the end-to-end storage deal process in Lotus,** falls short in terms of reliability and usability.
+
+Through various interactions with users and developers, we have confirmed these observations:
+
+1. users of Lotus struggle to understand the various deal stages and concepts.
+2. querying the network for status of an inflight deal is non-trivial, and requires too much protocol-specific knowledge.
+3. deal-making is a relatively long process, composed of various moving parts. When it fails midway, it's not easy to dig up the root cause.
+4. generally speaking, it is hard to strike successful deals on Filecoin mainnet due to a number of technical reasons that compound.
+
+Consequently, first-time users and devs who venture to experiment with Filecoin suffer an frictionful experience. Enough to turn away.
+
+We posit that **deal-making is the bread and butter of Filecoin**; *it is the reason the platform exists in the first place.* Thus it should work like a Swiss clock, and only fail exceptionally and/or for good reasons.
+
+> We decided to name this Project F3 in a wink to shooting to the same level of usability of Amazon S3 (albeit with notable differences).
+
+#### Assumptions & hypotheses
+
+_What must be true for this project to matter?_
+
+1. Users struggle to grasp the concepts and components that comprise the deal-making process end to end.
+2. Users don't have intuition of on what happens under the hood (not the details, but the major phases).
+3. New users desist, or end up frustrated.
```

Comment:
desist what ?

I think you mean "cease experimenting with Filecoin" ?

Reply:
Yep, synonym of "give up".